dockerExtension.ts

```diff
@@ -166,13 +152,40 @@ export async function activate(ctx: vscode.ExtensionContext): Promise<void> {
     activateLanguageClient(ctx);
 }
 
-export async function deactivate(): Promise<void> {
+function registerContainerCommands(): void {

```

Comment:
Personally I'd rather not break them up into separate categories - it just becomes a nuisance.  And what if you add new categories, or a command doesn't fit neatly into one or the other?  I regret doing that in the storage extension.

What I would like to see:
1) Move vscode-docker.createWebApp's code into a function
2) move all of the registerCommands into a single separate function

Reply:
Sounds good. I see where that comes from. 

Thanks. Will get on it.